### ISSUE TITLE
Something went wrong error

### ISSUE THREAD

**Issue by mizzunet** [abba23/spotify-adblock#52]
![image](https://user-images.githubusercontent.com/10193999/143034228-2ea3aa27-cbe5-448d-ba6a-175b5da98294.png)

```
~ LD_PRELOAD=/usr/local/lib/spotify-adblock.so spotify
spotify: /usr/lib/libcurl-gnutls.so.4: no version information available (required by spotify)
/opt/spotify/spotify: /usr/lib/libcurl-gnutls.so.4: no version information available (required by /opt/spotify/spotify)
/opt/spotify/spotify: /usr/lib/libcurl-gnutls.so.4: no version information available (required by /opt/spotify/spotify)
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
/proc/self/exe: /usr/lib/libcurl-gnutls.so.4: no version information available (required by /proc/self/exe)
[*] Error: No config file
[-] getaddrinfo:		dealer.spotify.com
[2021/11/23 19:05:49:2255] ERR: getaddrinfo failed: -4
[-] getaddrinfo:		dealer.spotify.com
[2021/11/23 19:05:49:8271] ERR: getaddrinfo failed: -4
[-] getaddrinfo:		dealer.spotify.com
[2021/11/23 19:05:51:2803] ERR: getaddrinfo failed: -4
[+] cef_urlrequest_create:	https://apresolve.spotify.com/?time=1637674549&type=accesspoint
[+] cef_urlrequest_create:	https://spclient.wg.spotify.com/gabo-receiver-service/public/v3/events/
[+] cef_urlrequest_create:	https://login5.spotify.com/v3/login
[-] getaddrinfo:		dealer.spotify.com
[2021/11/23 19:05:52:2827] ERR: getaddrinfo failed: -4
[*] Error: No config file
[-] getaddrinfo:		spclient.wg.spotify.com
[-] getaddrinfo:		apresolve.spotify.com
[-] getaddrinfo:		ap.spotify.com
[-] getaddrinfo:		ap.spotify.com
[-] getaddrinfo:		ap.spotify.com
[-] getaddrinfo:		ap.spotify.com
[-] getaddrinfo:		login5.spotify.com
[-] getaddrinfo:		ap.spotify.com
[+] cef_urlrequest_create:	https://login5.spotify.com/v3/login
[-] getaddrinfo:		login5.spotify.com
[+] cef_urlrequest_create:	https://login5.spotif
y.com/v3/login
[-] getaddrinfo:		login5.spotify.com
[+] cef_urlrequest_create:	https://login5.spotify.com/v3/login
[-] getaddrinfo:		login5.spotify.com
[+] cef_urlrequest_create:	https://login5.spotify.com/v3/login
[-] getaddrinfo:		login5.spotify.com
[-] getaddrinfo:		dealer.spotify.com
[2021/11/23 19:05:52:8845] ERR: getaddrinfo failed: -4
[+] cef_urlrequest_create:	https://login5.spotify.com/v3/login
[-] getaddrinfo:		login5.spotify.com
[-] getaddrinfo:		apresolve.spotify.com
[+] cef_urlrequest_create:	https://login5.spotify.com/v3/login
[-] getaddrinfo:		login5.spotify.com
[+] cef_urlrequest_create:	https://login5.spotify.com/v3/login
[-] getaddrinfo:		login5.spotify.com
[+] cef_urlrequest_create:	https://login5.spotify.com/v3/login
[-] getaddrinfo:		login5.spotify.com
[-] getaddrinfo:		dealer.spotify.com
[2021/11/23 19:05:54:3877] ERR: getaddrinfo failed: -4
[+] cef_urlrequest_create:	https://login5.spotify.com/v3/login
[-] getaddrinfo:		login5.spotify.com
[-] getaddrinfo:		ap.spotify.com
[-] getaddrinfo:		dealer.spotify.com
[2021/11/23 19:05:58:7486] ERR: getaddrinfo failed: -4
[+] cef_urlrequest_create:	https://login5.spotify.com/v3/login
[-] getaddrinfo:		login5.spotify.com
[+] cef_urlrequest_create:	https://login5.spotify.com/v3/login
[-] getaddrinfo:		login5.spotify.com
```

`spotify-adblock.so` taken from Releases and put to `/usr/local/lib/spotify-adblock.so`
Running Spotify normally works
Info:
Distro: Arch
Kernel: Linux 5.15.4-zen1-1-zen
Spotify version: 1:1.1.72.439-1

**Comment by abba23** [comment]
>[*] Error: No config file

Did you copy the `config.toml` to one of the locations mentioned in the readme?

**Comment by mizzunet** [comment]
No, I have not. Let me try again

**Comment by mizzunet** [comment]
Thank you very much, now it's working :+1:
